### ISSUE TITLE
[stable/airflow] airflow.config.KEY doesn't accept JSON strings

### ISSUE THREAD

**Issue by kristof-mattei** [helm/charts#23364]
**Describe the bug**
Chart fails to render when including a JSON string inside of `airflow.config.KEY`. 

When value is injected as follows:
```
AIRFLOW__KUBERNETES__WORKER_RESOURCES: "{ \"limit_cpu\": 1, \"limit_memory\": \"1Gi\", \"request_cpu\": 5, \"request_memory\": \"5Gi\" }"
```
I get this:
```
Error: UPGRADE FAILED: YAML parse error on airflow/templates/configmap-env.yaml: error converting YAML to JSON: yaml: line 45: did not find expected key
```

**Version of Helm and Kubernetes**:
Helm:
`version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"dirty", GoVersion:"go1.14.3"}`
K8s not applicable.

**Which chart**:
stable/airflow@7.3.0

**What happened**:
Chart failed to deploy.

**What you expected to happen**:
It didn't deploy

**How to reproduce it** (as minimally and precisely as possible):
Take chart, create `values.yaml` with the following
```
airflow:
    config:
        AIRFLOW__KUBERNETES__WORKER_RESOURCES: "{ \"limit_cpu\": 1, \"limit_memory\": \"1Gi\", \"request_cpu\": 5, \"request_memory\": \"5Gi\" }"
```

**Anything else we need to know**:
I think the issue is here:
https://github.com/helm/charts/blob/master/stable/airflow/templates/configmap-env.yaml#L78
If we remove the quotes around `{{ $option }}` it works.
Maybe we should replace it by `$option | quote`

**Comment by thesuperzapper** [comment]
Try using alternative YAML string syntax:
```yaml
airflow:
  config:
      AIRFLOW__KUBERNETES__WORKER_RESOURCES: |
        {
          "limit_cpu": 1,
          "limit_memory": "1Gi",
          "request_cpu": 5,
          "request_memory": "5Gi"
        }
```

**Comment by fabien-sarcel** [comment]
In my case, it's only work if I escape double quote:
```
airflow:
  config:        
    AIRFLOW__SECRETS__BACKEND_KWARGS: |
      {
        \"variables_file_path\": \"/var/airflow/metadata/var.json\", 
        \"connections_file_path\": \"/var/airflow/metadata/conn.json\"
      }
```

**Comment by thesuperzapper** [comment]
I have now fixed this issue in `7.10.0`
